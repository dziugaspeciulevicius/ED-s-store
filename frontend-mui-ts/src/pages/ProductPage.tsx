import React from "react";

const ProductPage = () => {
  return <div>SHOP</div>;
};

export default ProductPage;
