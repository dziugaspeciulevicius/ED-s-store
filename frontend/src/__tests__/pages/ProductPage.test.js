import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupTests";
import ProductPage from "../../pages/ProductPage.component";
jest.mock("../../pages/ProductPage.component");

describe("<ProductPage />", () => {
  it("Renders <ProductPage /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<ProductPage />)).toMatchSnapshot();
  });
});
