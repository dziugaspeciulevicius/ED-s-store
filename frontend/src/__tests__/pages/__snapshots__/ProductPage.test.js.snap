// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductPage /> Renders <ProductPage /> component correctly 1`] = `ShallowWrapper {}`;
